### DISCUSSION TITLE
powershell-7.4.3-linux-x64.tar.gz sha256sum mis-match

### DISCUSSION THREAD

**Discussion by rodmaccallum** [PowerShell/PowerShell#23990] [General]
I am not sure I am posting this in the correct place. I am getting a sha256sum mis-match with this tar.gz.

from hashes.sha256:
3ae8f3a556fd7e14fecefaacf15f5cfb0afab520b6f1aeae796d2eeafc9b6422 *powershell-7.4.3-linux-x64.tar.gz

from "SHA256 Hashes of the release artifacts" on github page:
powershell-7.4.3-linux-x64.tar.gz
    3ae8f3a556fd7e14fecefaacf15f5cfb0afab520b6f1aeae796d2eeafc9b6422

sha256sum powershell-7.4.3-linux-x64.tar.gz:
5cfcc228afd3ffce536ec4541abafe97c629afcd6dc85c9a20712894bbf65adb  powershell-7.4.3-linux-x64.tar.gz

**Comment by SteveL-MSFT** [comment]
Following-up on this

**Reply by rodmaccallum** [reply]
Yes, I am sorry, this was resolved a couple of weeks ago.
Thanks!
